packages/icons/docs/stories/helpers/constants.js

```diff
@@ -0,0 +1,13 @@
+export const LEFT_KEY = 37;
+export const UP_KEY = 38;
+export const RIGHT_KEY = 39;
+export const DOWN_KEY = 40;
+export const ESCAPE_KEY = 27;
+export const TAB_KEY = 9;
+export const ENTER_KEY = 13;
+export const DELETE_KEY = 8;
+export const SPACE_KEY = 32;
+export const PAGEUP_KEY = 33;
+export const PAGEDN_KEY = 34;
+export const END_KEY = 35;
+export const HOME_KEY = 36;
```

Comment:
here only use what is needed, we don't want to have stuff that never will be used when the use case emerge we will be adding more like this